src/runtime/httpd/WebCloud/SMRUCC.HTTPInternal/Core/HttpProcessor.vb

```diff
@@ -174,7 +174,9 @@ Namespace Core
 
             ' we probably shouldn't be using a streamwriter for all output from handlers either
             ' 2017-3-25 使用utf8来尝试解决中文乱码问题
-            outputStream = New StreamWriter(New BufferedStream(socket.GetStream()), Encoding.UTF8)
+            outputStream = New StreamWriter(New BufferedStream(socket.GetStream()), TextEncodings.UTF8WithoutBOM) With {
+                .NewLine = vbCrLf
```

Comment:
Http header new line is VBCrLf, orelse some http client like CURL will not working for parsing header with other newline value